packages/integration/lib/src/views/activity/activity_screen.dart

```diff
@@ -398,6 +398,7 @@ class _LMFeedActivityScreenState extends State<LMFeedActivityScreen> {
                       LMFeedFireAnalyticsEvent(
                         eventName: LMFeedAnalyticsKeys.postDeleted,
                         deprecatedEventName: LMFeedAnalyticsKeysDep.postDeleted,
+                        widgetSource: LMFeedWidgetSource.activityScreen,
```

Comment:
Let's clear this with product once

Reply:
We haven't modified any key or property map defined by product.